src/main/drivers/rangefinder/rangefinder_virtual.c

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Is this bridge even needed? It only fills some rangefinder parameters, and that parameters are device-specific. It will break if you connect different rangefinder over MSP